--- a/pru_sw/app_loader/interface/prussdrv.c	2013-08-20 10:00:44.812458775 +0000
+++ b/pru_sw/app_loader/interface/prussdrv.c	2013-08-20 10:02:04.573912410 +0000
@@ -98,7 +98,7 @@
     switch (prussdrv.version) {
     case PRUSS_V1:
         {
-            printf("AM18XX\n");
+ //           printf("AM18XX\n");
             prussdrv.pru0_dataram_phy_base = AM18XX_DATARAM0_PHYS_BASE;
             prussdrv.pru1_dataram_phy_base = AM18XX_DATARAM1_PHYS_BASE;
             prussdrv.intc_phy_base = AM18XX_INTC_PHYS_BASE;
@@ -112,7 +112,7 @@
         break;
     case PRUSS_V2:
         {
-            printf("AM33XX\n");
+ //           printf("AM33XX\n");
             prussdrv.pru0_dataram_phy_base = AM33XX_DATARAM0_PHYS_BASE;
             prussdrv.pru1_dataram_phy_base = AM33XX_DATARAM1_PHYS_BASE;
             prussdrv.intc_phy_base = AM33XX_INTC_PHYS_BASE;
@@ -652,7 +652,7 @@
     if (fPtr == NULL) {
         printf("File %s open failed\n", filename);
     } else {
-        printf("File %s open passed\n", filename);
+ //       printf("File %s open passed\n", filename);
     }
     // Read file size
     fseek(fPtr, 0, SEEK_END);
